.changeset/violet-tools-refuse.md

```diff
@@ -0,0 +1,21 @@
+---
+"@fluidframework/container-runtime": minor
+"@fluidframework/core-interfaces": minor
+"@fluidframework/datastore": minor
+"@fluidframework/runtime-utils": minor
+"@fluidframework/shared-object-base": minor
+"@fluid-private/test-dds-utils": minor
+"@fluid-private/test-end-to-end-tests": minor
+"@fluidframework/test-runtime-utils": minor
+"@fluid-experimental/tree": minor
+"__section": deprecation
+---
+Deprecate IFluidHandleInternal.bind function
```

Comment:
Nit:
```suggestion
Deprecate IFluidHandleInternal.bind
```

Reply:
(I'm recommending we try to keep the name of the affected API as close to the start of changesets as possible - makes it easier to scan release notes for relevant entries)